<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addExpenseY</class>
 <widget class="QDialog" name="addExpenseY">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>386</width>
     <height>109</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Category</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Amount</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Month</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QComboBox" name="comboBox_3">
          <item>
           <property name="text">
            <string>Life Insurance</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Car Insurance</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Health Insurance</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Gifts</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Clothing</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Home/Renovations</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Student Loans</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Tuition Fee</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit_2"/>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_4">
          <item>
           <property name="text">
            <string>January</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>February</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>March</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>April</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>May</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>June</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>July</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>August</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>September</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>October</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>November</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>December</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
